### ISSUE TITLE
Did you train it based on the resnet-101 imagenet-classification model?

### ISSUE THREAD

**Issue by wanggrun** [DrSleep/tensorflow-deeplab-resnet#159]
You said that "the performance score on the validation set has increased to 80.1%". Do you mean that the model is trained based on the resnet-101 imagenet-classification model? Or just finetuned from the deeplab's released model?

**Comment by DrSleep** [comment]
please read the full sentence: 

> Fixed the evaluation procedure: the 'void' label (255) is now correctly ignored. As a result, the performance score on the validation set has increased to 80.1%

All the models are the ones released by the DeepLab team
